### DISCUSSION TITLE
Quassar or Vuetify for our open-source project ?

### DISCUSSION THREAD

**Discussion by rizkiheryandi** [quasarframework/quasar#10865] [Show and Tell]
Hello friends,

Im rizki from [uasoft](https://github.com/uasoft-indonesia) and maintainer for [badaso](https://github.com/uasoft-indonesia/badaso)

Badaso is a open-source headless low code development platform based on PHP and Vue, we can build application 10x faster. 

So, we are planning to start develop version 3.x of badaso

In previous version of badaso (v2.x) we are using "vuesax", but looks like it's already abandoned project

So in the v3.x we will replace vuesax with other well maintained vue component like quassar & vuetify

Because we already use vuetify previously, so we plan to use vuetify before

But as mentioned here https://github.com/uasoft-indonesia/badaso/discussions/394 

We want the new badaso has support vue 3, but vuetify doesnt has stable version for vue3 for now

So we found quassar already support vue 3 as stable version, because of that, quassar better meet the criteria

So i just wanna to know from other opinion, so we can choose wisely

Thank you in advance :) hope your days great !

(Btw, thank you for the team of quassar, quassar is awesome ! )

**Comment by smolinari** [comment]
@rizkiheryandi - You say your platform is based on PHP. Is PHP purely used for generating an API? Or are you serving up HTML pages on the server via PHP too? Depending on what you reply will change the answer I could give somewhat. 

Scott

**Reply by rizkiheryandi** [reply]
Hello @smolinari 

Yes sure, PHP purely for API and the frontend is a SPA using vue

**Reply by smolinari** [reply]
@rizkiheryandi - Thanks for answering. Next question, do you want to do more with your code then possibly just a SPA? Is maybe building out to other devices/ platforms in your plans or a possibility you might face at some point in the future?

Scott

**Reply by rizkiheryandi** [reply]
Yes sure, our tools should also support PWA & electron

**Reply by smolinari** [reply]
@rizkiheryandi - Ok. Great. And this is where Quasar will make your life much, much easier. With Vuetify, you'll be responsible for wiring up the build processes for PWA and Electron builds. Plus, if anything goes wrong, Vuetify probably won't be supporting you as they don't have any responsibility over any build process. It's only a component library (it's not even a framework). 

The Quasar team will care, if your build process is having issues, as the Quasar CLI covers those processes and it does them well (i.e. with best-practices and securely). 

I made an image that might also show what you are getting and the major difference between Quasar and "the rest". 

![image](https://user-images.githubusercontent.com/3314957/136317902-a600064f-c1f3-4e12-b9b5-23cba3923b3c.png)

Scott

**Reply by rizkiheryandi** [reply]
Nice, thank you for the all valuable informations scott @smolinari

**Comment by zanzara** [comment]
@rizkiheryandi We are here in Quasar github-repo.  Of course we only recommend to use Quasar! What else? ;-)
Go to vuetify and they will recommend theirs.
Seriously: go for Q if you need Vue3 support yet.
Apparently to me, you use PHP only as backend-API  - I also did this once even using Laravel7  =:0
(But nowadays I'm lucky using node.js&co as backend)

**Reply by rizkiheryandi** [reply]
Hello @zanzara 

Yeah youre right, we already choose quassar, because we wanna fully use new tech in the next version, included the vue 3.

Nodejs is good, we also use nodejs in our another project

thankyou
